docs/UserGuide.md

```diff
@@ -95,6 +96,23 @@ Format: `view INDEX`
 Examples:
 * `view 1` shows the 1st flashcard (in the displayed flashcard list).
 
+### Finding cards : `find`
+
+Find cards according to search criteria containing any of the given keywords.
+
+Format: `find CRITERIA KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `equation` will match `Equation`
+* The order of the keywords does not matter. e.g. `Newton Equation` will match `Equation Newton`
+* The search will be according to the `CRITERIA` given. `q/` to search by questions or `c/` to search by categories.
```

Comment:
Perhaps you should also state that the user can search by specifying both the question and category. eg. "find q/ <keyword> c/ <keyword>".

Reply:
It is okay, the user will not be able to search by multiple criteria, it will be either by question or by category.